### ISSUE TITLE
Custom Loss Function | Keras | Tensorflow backend

### ISSUE THREAD

**Issue by SwagatDas** [keras-team/keras#12965]
<em>I am trying to write a custom loss function in Keras (Tensorflow backend). The loss function has to use the instantaneous training input (features, not y_true) of the current mini-batch. How can I incorporate the current instantaneous batch's training features into my loss function? </em>

**Comment by LHillmann** [comment]
Answered here: [https://stackoverflow.com/questions/43818584/custom-loss-function-in-keras](stackoverflow). Please close the issue if this answers your question. 

Best,
Leon
